packages/db/src/types.ts

```diff
@@ -83,3 +83,25 @@ export interface PostgresProviderRecord extends PostgresProvider {
   updated_at: number,
   cumulusId: number,
 }
+
+export interface PostgresRule {
+  name: string,
+  workflow: string,
+  collection_cumulus_id: number,
```

Comment:
Collection/provider aren't required in Dynamo, so I don't think it's intentional/correct to make them required in RDS

Reply:
I see. I thought it was required in RDS because it's `nonNullable`
```
    table
      .integer('collection_cumulus_id')
      .references('cumulus_id')
      .inTable('collections')
      .notNullable();
    table
      .integer('provider_cumulus_id')
      .references('cumulus_id')
      .inTable('providers')
      .notNullable()
```
https://github.com/nasa/cumulus/blob/feature-rds/lambdas/db-migration/src/migrations/20201014105024_create_rules_table.ts#L17-L25

Reply:
Yeah, I think that table schema was defined erroneously. And fun fact: I'm the one who did it! See #1932 

But providers/collections are not set as required in the Dynamo schema, so I don't know why they suddenly would be for RDS.

Reply:
Haha okay, I will update it so they are no longer required